plugins/notifications-backend/src/service/router.ts

```diff
@@ -23,7 +25,14 @@ import { RouterOptions } from './types';
 export async function createRouter(
   options: RouterOptions,
 ): Promise<express.Router> {
-  const { logger, dbConfig, catalogClient } = options;
+  const { logger, database, discovery, config } = options;
+
+  // workaround for creating the database when client is not sqlite
+  const dbClientWorkaround = await database.getClient();
```

Comment:
please don't call it workaround. it only makes people notice it. it is what it is. a db client

Reply:
We have actual `dbClient` below. So renaming